### PR TITLE
Documenting Authentication Resource 

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. updated Auth response with seperate header
2. added javascript examples
3. added some dont's which result in 500 internal server error status

### CODE REVIEW

File: source/includes/auth.md [hunk 1]
```diff
@@ -156,7 +275,26 @@ GET /openmrs/ws/rest/v1/location?tag=Login+Location'
...
+myHeaders.append("Cookie", "JSESSIONID=2D158E83ACFB788998C7DB495F07C1B9");
+
+var requestOptions = {
+  method: 'GET',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+
+fetch("https://demo.openmrs.org/openmrs/ws/rest/v1/location?tag=Login+Location'\n", requestOptions)
+  .then(response => response.text())
```
Comment (dino-saurabh) [inline source/includes/auth.md:291]: @Akayeshmantha  what is this API call supposed to return since I get a success response but it is empty.

**Comment by dino-saurabh** [comment]
@Akayeshmantha the Authentication resource was pretty clear with conscise descriptions I have added few dont's where I got stuck while using this endpoint. please review !!

File: source/includes/auth.md (hunk 1 above)
Comment (Akayeshmantha) [reply source/includes/auth.md:291]: It should be the login location

**Review by Akayeshmantha — APPROVED** [review]
LGTM

**Pull request merged**